engineering/methodology.md

```diff
@@ -1,122 +1,90 @@
 # Engineering methodology
 
-Date: 2018-04-12
+Date: 2018-11-29
 
 * [Intro](#intro)
-* [History](#History)
-* [Objectives and Key Results (OKRs)](#objectives-and-key-results-okrs)
-* [Kanban](#kanban)
-  * [Company Kanban](#company-kanban)
-  * [Team Kanban](#team-kanban)
-    * [How to pick next issues to worked on?](#how-to-pick-next-issues-to-work-on)
+* [OKRs](#okrs)
+* [Backlog and Features-Idea repositories](#backlog-and-features-idea-repositories)
 * [Leads Meeting](#leads-meeting)
-* [Team Meeting](#team-meeting)
+* [Team Sync Meeting](#team-sync-meeting)
 * [Standup](#standup)
 * [Demo](#demo)
-* [Keeping track of things](#keeping-track-of-things)
-* [Definitions](#definitions)
-  * [What does TODO mean?](#what-does-todo-mean)
-  * [What does Done mean?](#what-does-done-mean)
-  * [What does Blocker mean?](#what-does-blocker-mean)
+* [Retrospective Meeting](#retrospective-Meeting)
 
 ## Intro
 
-This document describes the current development methodology followed by engineering.
+This methodology's an evolution of [previous workflows](https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md) and experiences, and it's on constant evolution an open to change at any moment.
 
-We use simple, company and team-level Kanban boards,
-so that all iterations including planning/work/retrospective do not happen
-synchronously at the same weeks for all the teams.
+The current workflow is based on this main concepts:
 
-Meetings for high-level sync including product direction, vision, etc are
-proactively scheduled by each Team Lead as needed. All Leads have periodical
-hands-on meetings with the CTO.
+* OKRs, company/team engineering OKRs are what should drive every single decision taken by the Engineering Team.
+* A [Workflow](workflow.md), that defines the process of definition and development of the tasks, from the idea to the release.
+* A Backlog and Features-Idea repositories, where the approved tasks and the suggestions are.
+* Leads meeting, a meeting to align all efforts done by the different teams at engineering.
```

Comment:
```suggestion
* Leads meeting to align all efforts done by the different teams at engineering every week.
```

Reply:
The periodicity is removed intentionally from the guides.